sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeviceUpdateClientSample/Sample.cs

```diff
@@ -134,49 +132,81 @@ public async Task RunAsync()
         private async Task<string> ImportUpdateStepAsync(string version)
         {
             ContentFactory contentFactory = new ContentFactory(_connectionString, BlobContainer);
-            ImportUpdateInput update = await contentFactory.CreateImportUpdate(SimulatorProvider, SimulatorModel, version);
+            var update = await contentFactory.CreateImportUpdate(SimulatorProvider, SimulatorModel, version);
```

Comment:
This variable is not used in the sample - why is it needed?

Reply:
removed used variable